BackendApartmentReservation/BackendApartmentReservation/Repositories/EmployeeRepository.cs

```diff
@@ -28,6 +28,11 @@ public async Task<DbEmployee> GetEmployeeById(int employeeID)
             return await _context.Employees.FindAsync(employeeID);
         }
 
+        public async Task<DbEmployee> GetEmployeeById(string employeeID)
```

Comment:
Name it `GetEmployeeByEmployeeId`, to make it explicit.

And can you look by non-key field?